core/trino-main/src/main/java/io/trino/cost/TableScanStatsRule.java

```diff
@@ -37,6 +38,7 @@
 public class TableScanStatsRule
         extends SimpleStatsRule<TableScanNode>
 {
+    private static final double UNKNOWN_NULLS_FRACTION = 0.1;
```

Comment:
We probably should use same constant as in `getOutputSizeForSymbol`

Reply:
That assumes zero nulls, I don't want to assume that to avoid estimating `IS NULL` filter to be 0.
That method will anyway operate on top of estimates provided from here, so there is no need to change the logic there.